### DISCUSSION TITLE
Not found Plots Caution,

### DISCUSSION THREAD

**Discussion by Bone117** [swar/Swar-Chia-Plot-Manager#539] [Q&A]
**Describe the bug**
Error message： Not found Plots Caution, deleting these plots will delete them forever. Check that the storage devices are properly connected.
when I use the official GUI this can work, and the plot file generated by the command line cannot be recognized

- OS: [ win10]

![image](https://user-images.githubusercontent.com/35058785/118602419-3df82b00-b7e5-11eb-9b5e-6def5c824f9a.png)
![image](https://user-images.githubusercontent.com/35058785/118603393-6cc2d100-b7e6-11eb-8ba8-b920b8f595f9.png)

please help me ，thanks！

**Comment by rdprosser** [comment]
I have the same issue... my plots generated with SWAR are not being recognized by the GUI for farming...

**Reply by rdprosser** [reply]
I don't know if this is anyone elses issue, but my problem stemmed from inputting my key into the config file. It should have still worked but it did not.... Once I left the public key portion blank it worked without issue

**Comment by billyrsp5199** [comment]
i have same problem
